src/Clipboard.svelte

```diff
@@ -1,15 +1,23 @@
 <script lang="ts">
   import Icon from '@app/Icon.svelte';
   import { toClipboard } from '@app/utils';
+  import { createEventDispatcher } from "svelte";
+
+  const dispatch = createEventDispatcher();
+
+  const copy = () => {
+    toClipboard(text);
+    dispatch("copied");
+  };
 
   export let text: string;
   export let small = false;
 </script>
 
 <style>
   .clipboard {
-    width: 32px;
-    height: 32px;
+    width: 30px;
+    height: 30px;
```

Comment:
What's this about? I try to stick to multiples of 1rem (16px)

Reply:
Yeah, the issue here is that the `input` field has a height of 30px so when put the 32px high span element on top of it, the span element overlaps on both sides of the `input` field.
<img width="162" alt="Bildschirmfoto 2022-08-17 um 07 55 50" src="https://user-images.githubusercontent.com/7912302/185045054-2a6aa3cb-603c-4fe6-9ad8-3ae48e5f04f6.png">

By reducing the span height to 30px it matches the height of the input field.

Reply:
Hmm or we could also increase the input field from 30px to 2rem 🤔

Reply:
Yeah the input field may aswell be rem-aligned since it's so close anyway. Can we fix this in a different PR and leave the button as-is for now?

Reply:
> Yeah the input field may aswell be rem-aligned since it's so close anyway. Can we fix this in a different PR and leave the button as-is for now?

Ok yeah no problem

Reply:
@cloudhead sizes have been reverted to multiples of 1rem in [904f58a](https://github.com/radicle-dev/radicle-interface/pull/291/commits/904f58adc23d21c93ce3143fddd767208b33856b)